packages/fxa-auth-server/test/client/index.js

```diff
@@ -253,7 +253,7 @@ module.exports = config => {
             c.authAt = data.authAt;
             c.verified = data.verified;
             c.verificationReason = data.verificationReason;
-            c.verificationMetho = data.verificationMethod;
```

Comment:
The type checker found this amongst lots of probably erroneous bits. This seemed pretty obviously a mistake.

Reply:
Oof, guess we weren't really checking this.